pom.xml

```diff
@@ -322,6 +368,60 @@
                 <version>${gitflow-incremental-builder.version}</version>
             </extension>
         </extensions>
+
```

Comment:
Just a question, why do we need to ignore below executions of 2 plugins for Eclipse?